atm.js

```diff
@@ -1,3 +1,91 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
 });
+
+checkingDeposit = $("#checking").find('.deposit');
+checkingWithdraw = $("#checking").find('.withdraw');
+savingsDeposit = $("#savings").find('.deposit');
+savingsWithdraw = $('#savings').find('.withdraw');
+checkingBalance = 0;
```

Comment:
I'd move the above 4 lines inside the document-ready anonymous call-back block. Sometimes, the page won't have finished loading and $ attempts to select an element that hasn't been rendered to the page yet. 

You have your script tag at the bottom, which mostly ensures this won't happen, but doc.ready callbacks are more reliable.

Also use keyword `var` for declaring variables to avoid the use of declaring globals.